highbury/assetlist.json

```diff
@@ -0,0 +1,120 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "highbury",
+  "assets": [
+    {
+      "description": "The native staking and governance token of the Blackfury ecosystem",
+      "denom_units": [
+        {
+          "denom": "ufury",
+          "exponent": 0
+        },
+        {
+          "denom": "fury",
+          "exponent": 6
+        }
+      ],
+      "base": "ufury",
+      "name": "Fury",
+      "display": "fury",
+      "symbol": "FURY",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/highbury/images/fury.svg",
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/highbury/images/fury.png"
+      },
+      "coingecko_id": "fury"
```

Comment:
This cgid doesn't exist. Please remove

Reply:
This entry has been revised with the correct cgid - "fanfury".